plutus-playground-server/app/Webserver.hs

```diff
@@ -79,9 +83,17 @@ app handlers _staticDir githubEndpoints config =
       simpleCorsResourcePolicy
         {corsRequestHeaders = ["content-type", "set-cookie"]}
 
+startStatsd :: MonadIO m => m Store
+startStatsd = liftIO $ do
+    store <- newStore
+    void $ forkStatsd defaultStatsdOptions store
+    pure store
+
 run :: (MonadLogger m, MonadIO m) => Settings -> FilePath -> Config -> m ()
 run settings _staticDir config = do
+  store <- startStatsd
   githubEndpoints <- liftIO Auth.mkGithubEndpoints
   handlers <- PS.mkHandlers
+  appMonitor <- liftIO $ monitorEndpoints (Proxy @PA.API) store
   logInfoN "Starting webserver."
-  liftIO . runSettings settings $ app handlers _staticDir githubEndpoints config
+  liftIO . runSettings settings . appMonitor $ app handlers _staticDir githubEndpoints config
```

Comment:
Fair bunch of duplication between the two files but I'll let you deal with it when it's annoying.